src/tallies/tally_scoring.cpp

```diff
@@ -1374,15 +1374,15 @@ score_general_mg(const Particle* p, int i_tally, int start_index,
     p_g = p->g_;
   }
 
-  // To significantly reduce de-referencing, point matxs to the macroscopic
-  // Mgxs for the material of interest
-  data::mg.macro_xs_[p->material_].set_angle_index(p_u);
+  // For shorthand, assign pointers to the material and nuclide xs set
+  Mgxs* nuc_xs = &data::mg.nuclides_[i_nuclide];
+  Mgxs* macro_xs = &data::mg.macro_xs_[p->material_];
```

Comment:
Why not use a reference for these?
```
auto& nuc_xs = data::mg.nuclides_[i_nuclide];
auto& macro_xs = data::mg.macro_xs_[p->material_];
```
and then use `.` access below rather than `->`.

Reply:
No particular reason, fixed.